source/blog/2018-07-13-the-ember-times-issue-55.md

```diff
@@ -0,0 +1,136 @@
+
+---
+
+## [Don't Worry, Ember CLI Got You Covered](https://github.com/ember-cli/ember-cli-update) 🖳
+The **number one** tool for updating Ember.js apps or addons just got **even better**. The [newest version](https://github.com/ember-cli/ember-cli-update/releases) of `ember-cli-update` now runs `qunit-dom-codemod` for you. This means that you, with close to no effort at all, can utilize this great addon for your tests.
+
+And the cool things is that `ember-cli-update` fetches new codemods **during runtime** - so no need to update to get this nice codemod! To run the codemod just type `ember-cli-update --run-codemods` and magic will take care of the rest for you.
+
+You can visit the addon page for [ember-cli-update](https://github.com/ember-cli/ember-cli-update) to read all about it or visit [qunit-dom](https://github.com/simplabs/qunit-dom) to learn about all the nice features it provides.
+
+---
+
+## [Start Small - How I Became an Ember Addon Maintainer by Ryan Mark](github.com/tylerturdenpants) 💪
+
+[Ember User Activity](https://github.com/elwayman02/ember-user-activity/) is a great addon.  It fit my needs and worked perfectly...until I needed to use **Fastboot**. Seeing how others had solved Fastboot related compatibility issues, I cloned [`ember-user-activity`](https://github.com/elwayman02/ember-user-activity/) and got to work. After a few suggested tweaks by the author, my PR was merged, and a new version was released.  A few weeks later, a new version of ember-cli was released. I've always updated my own app, but I had never done an addon.  Excited about this new **challenge**, I updated [`ember-user-activity`](https://github.com/elwayman02/ember-user-activity/) to the latest ember-cli version. Moments later, after submitting my PR, the author sent me an invitation to **collaborate**. The author asked if I could **help** him maintain the addon, and I **enthusiastically** agreed. As of today, I have published a handful of releases. Even though I didn't create the addon, I had **ownership**, and it was exciting to be a part of the addon **community**.
+
+TL;DR, one of the easiest ways to contribute, and possibly becoming a collaborator/maintainer of an addon, is doing small, chore like work for the author. Authors, and even current maintainers, always appreciate the help. Not only does it keep the addon fresh, but often the addon needs a fix or two to be compatible, allowing for even more contributing.
+
+Interested in contributing in a hurry? According to [@mixonic](https://twitter.com/mixonic), an Ember.js core team member, the version of Ember.js used in [TodoMVC](http://todomvc.com/examples/emberjs/) has been out of date and he's asking for help. Check out the [conversation](https://github.com/tastejs/todomvc/pull/1790) and lend a helping hand!
```

Comment:
I think it would be 

`an Ember.js Core team member`

according to the new [org update](https://emberjs.com/blog/2018/06/30/organizing-our-contributors.html).